### ISSUE TITLE
Blank pdf page

### ISSUE THREAD

**Issue by delcullu** [eKoopmans/html2pdf.js#517]
I m trying to get pdf output but getting 1 blank page only. I couldnt findout why.

**Comment by delcullu** [comment]
[form.zip](https://github.com/eKoopmans/html2pdf.js/files/7957896/form.zip)

Could anyone help plz?

**Comment by delcullu** [comment]
[form.zip](https://github.com/eKoopmans/html2pdf.js/files/7957950/form.zip)

This is the file that i m working. Plz help.

**Comment by simonpeters** [comment]
Having the same issue

**Comment by suresh-ramasamy002** [comment]
Facing the same issue with Chrome(Latest Version) but it is working good all other browsers but, mostly people are using chrome browser. Anybody help me out with this

**Comment by delcullu** [comment]
#googoose-cover {
                position: absolute; 
                margin-bottom: 20mm;
                top: 150px;
                z-index: -1;
                width: 100%;
                text-align: justify;
            }

Position absolute to relative solve my issue

**Comment by simonpeters** [comment]
what do you mean @delcullu ?

**Comment by suresh-ramasamy002** [comment]
@simonpeters, i think he might have different issues with the css only. but our blank page is completely different

**Comment by delcullu** [comment]
May be the reason that u use css position or margins is reveals that issue. I tried find which code was responsible for that. I deleted some css codes and tried many times so i figured this is about position:absolute line that cause my problem.

**Comment by delcullu** [comment]
Try to start with simple text export to pdf to find it out.

**Comment by rajathegde98** [comment]
Try to remove the Grammarly extension, jsPdf has some issues while generating the PDF when using Grammarly.

**Comment by simonpeters** [comment]
with me its happening because I'm using ionic and it seems the default css it adds makes html2pdf not render anything.

**Comment by rohmansca** [comment]
> Try to remove the Grammarly extension, jsPdf has some issues while generating the PDF when using Grammarly.

Yes, It solves the issue. When I use a different browser that doesn't have Grammarly, It's going well.
Thanks, @rajathegde98

**Comment by bengsiswantoh** [comment]
> with me its happening because I'm using ionic and it seems the default css it adds makes html2pdf not render anything.

@simonpeters  try to install latest html2canvas maybe you have same [problem](https://github.com/eKoopmans/html2pdf.js/issues/532) with me

**Comment by hamidhomapour** [comment]
the following worked for me:

1. `npm install marked html2pdf.js`
2. `npm install --save-dev @types/marked`
3. 

```
import React from 'react';
import { marked } from 'marked';
// @ts-ignore
import * as html2pdf from 'html2pdf.js';
```
